.bundle/config

```diff
@@ -0,0 +1,3 @@
+---
+BUNDLE_WITHOUT: omnibus_package
+BUNDLE_FROZEN: '1'
```

Comment:
Do we need to explicitly set BUNDLE_FROZEN or can we use a --deployment flag when needed?
Per the docs: BUNDLE_FROZEN defaults to true when --deployment is used.